subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -12,133 +12,112 @@
+flexible enough to build almost any type of software. Gradle bases its design on the following fundamentals:
 
 High performance::
-Gradle avoids unnecessary work by only running the tasks that need to run because their inputs or outputs have changed. You can also use a build cache to enable the reuse of task outputs from previous runs or even from a different machine (with a shared build cache).
-+
-There are many other optimizations that Gradle implements and the development team continually work to improve Gradle's performance.
+Gradle avoids unnecessary work by only running tasks that need to run because inputs or outputs changed.
+Gradle uses various caches to reuse outputs from previous runs. With a shared build cache, you can even reuse outputs across multiple machines.
 JVM foundation::
-Gradle runs on the JVM and you must have a Java Development Kit (JDK) installed to use it. This is a bonus for users familiar with the Java platform as you can use the standard Java APIs in your build logic, such as custom task types and plugins. It also makes it easy to run Gradle on different platforms.
-+
-Note that Gradle isn't limited to building just JVM projects, and it even comes packaged with support for building native projects.
+Gradle runs on the JVM. This is a bonus for users familiar with Java, since build logic can use the standard Java APIs.
+It also makes it easy to run Gradle on different platforms.
 Conventions::
-Gradle takes a leaf out of Maven's book and makes common types of projects — such as Java projects — easy to build by implementing conventions. Apply the appropriate plugins and you can easily end up with slim build scripts for many projects. But these conventions don't limit you: Gradle allows you to override them, add your own tasks, and make many other customizations to your convention-based builds.
+Gradle makes common types of projects easy to build through conventions. Plugins set sensible defaults to keep build scripts minimal.
+But these conventions don't limit you: you can configure settings, add your own tasks, and make many other customizations in your builds.
 Extensibility::
-You can readily extend Gradle to provide your own task types or even build model. See the Android build support for an example of this: it adds many new build concepts such as flavors and build types.
+You can readily extend Gradle to provide your own build logic with custom tasks and plugins.
+See Android builds for an example: they add many new build concepts such as flavors and build types.
 IDE support::
-Several major IDEs allow you to import Gradle builds and interact with them: Android Studio, IntelliJ IDEA, Eclipse, and NetBeans. Gradle also has support for generating the solution files required to load a project into Visual Studio.
+Several major IDEs enable imports and interaction with Gradle builds, including Android Studio, IntelliJ IDEA, Eclipse, and NetBeans.
```

Comment:
I don't think "enable imports" sounds right, I think the original phrase was better here.

Reply:
Should we mention VSCode too?  What is the current state of NetBeans support?

Reply:
```suggestion
Several major IDEs provide interaction with Gradle builds, including Android Studio, IntelliJ IDEA, Eclipse, VSCode, and NetBeans.
```
Does this sound better to you? As far as I know, there are still supported NetBeans versions that work with supported Gradle versions... but I haven't tested it personally. Open to removing it if you think that it's better that way.